src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -733,6 +734,7 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         pass_config->disable<ngraph::pass::TransposeReduction>();
         // Operations Max and Min aren't supported
         pass_config->disable<ngraph::pass::ConcatReduceFusion>();
+        pass_config->disable<ngraph::pass::SoftSignDecomposition>();
```

Comment:
Added.

Reply:
...and reverted ([6857a7d](https://github.com/openvinotoolkit/openvino/pull/11546/commits/6857a7d4004b8d680bccedd295cc0b022bee0755)), because it failed;)

I think we shouldn't disable SoftSign decomposition for GNA without functional tests. 
Also frontends still produce already decomposed SoftSign, so we don't have validation of models with "not decomposed" SoftSign within this PR.

I would suggest to move GNA plugin changes to a separate PR.
Is GNA SoftSign shell still needed, with support of SoftSign in ov core? 
Should usage of `ov::intel_gna::op::SoftSign` be updated to `ov::op::v9::SoftSign`?
@openvinotoolkit/openvino-ie-gna-maintainers, @sirzabek 
https://github.com/openvinotoolkit/openvino/blob/35b8972da38db2fd6fe53552a2cb99eb06e78e3e/src/plugins/intel_gna/ops/softsign.hpp#L11-L19
https://github.com/openvinotoolkit/openvino/blob/35b8972da38db2fd6fe53552a2cb99eb06e78e3e/src/plugins/intel_gna/transformations/substitute_softsign.cpp#L25

Reply:
As agreed with @jiwaszki SoftSign changes for GNA plugin will be added in a separate PR. I'd leave internal GNA representation of SoftSign for now, the full switch to nGraph hasn't been accomplished yet. And even when it's done, there will be some internal representation for GNA required anyway.